src/BuildScriptGenerator/DefaultRunScriptGenerator.cs

```diff
@@ -123,6 +123,12 @@ public string GenerateBashScript(RunScriptGeneratorContext ctx)
 #pragma warning restore CA2201
             }
 
+            BuildConfigurationFIle buildConfigFile = BuildConfigurationFIle.Create(ctx.SourceRepo.ReadFile("appsvc.yaml"));
+            if (!string.IsNullOrEmpty(buildConfigFile.Run))
+            {
+                File.AppendAllText(this.tempScriptPath, Environment.NewLine + buildConfigFile.Run);
+            }
+
```

Comment:
@waliMSFT friendly reminder to add integration tests for this in: https://github.com/microsoft/Oryx/tree/main/tests/Oryx.Integration.Tests/DotNetCore